components/script/dom/webglrenderingcontext.rs

```diff
@@ -71,6 +71,20 @@ macro_rules! handle_object_deletion {
     };
 }
 
+macro_rules! object_binding_to_js {
```

Comment:
nit: Maybe calling this `object_binding_to_js_or_null`, even if a bit more annoying, would allow not having to come here to see what the behavior is when it's `None`? It's a uber-nit though, and sort of opinionated, so I wouldn't gate landing this on it.